### DISCUSSION TITLE
Styling crashes on reload

### DISCUSSION THREAD

**Discussion by B4CKSL4SH-44** [neuefische/questions#355] [Web] [answered]
Hello there,
maybe someone already encountered this issue: on local, everything seemed to work nice, but when you go on vercel and reload the page, the whole styling crashes. Can someone help me?

here's the [Vercel Link](https://capstone-tropicolony-git-levelstructure-simon-schuster44.vercel.app/level1)
And here's my [Repo (branch LevelStructure)](https://github.com/simon-schuster44/Capstone-Tropicolony/tree/LevelStructure)

**Comment by pixelass** [comment]
Can you send a screenshot of the expected and actual appearance?

I cannot reproduce the issue. On reload everything still looks the same


![Screen Shot 2022-12-01 at 10 43 26](https://user-images.githubusercontent.com/1148334/205019687-1a7cd743-dc04-4fb1-8313-a57b37fd2e17.png)

**Reply by B4CKSL4SH-44** [reply]
![Bildschirmfoto 2022-12-01 um 10 45 29](https://user-images.githubusercontent.com/115007480/205020195-4a2e9969-5c97-438c-bace-0b07b143692b.png)
Well, that's how it's supposed to look...

**Comment by B4CKSL4SH-44** [comment, marked as answer]
I just realized that I used both CSS and styled components. Maybe that was a mistake, 'cause after reload only the CSS styling seems to work...

**Reply by pixelass** [reply]
You shouldn't use babel. there might be a conflict with babel. If you need babel you have to add extra steps in your setup. I'm not sure if that's the root cause of the issue but it's worth investigating.

I can see that you have the babelrc file for `inline-react-svg`. You can also just use the svg files directly in react or see if `inline-react-svg` supportst SWC

```jsx

function MySvg() {

  return <svg><path d="…"/></svg>
}
```

**Reply by pixelass** [reply]
Also, you shouldn't mix CSS files with styled-components. It should work but is an anti-pattern.

**Reply by pixelass** [reply]
I also found this issue: https://github.com/vercel/next.js/discussions/33161

Maybe it helps when moving away from babel

**Reply by pixelass** [reply]
I also found this issue: https://github.com/vercel/next.js/discussions/33161

Maybe it helps when moving away from babel
